.github/workflows/test.yaml

```diff
@@ -105,14 +105,13 @@ jobs:
           - docker
           - nerdctl
           - kind
+          - auto # auto runtime: will detect runtime automatically
```

Comment:
``` yaml
kind: KwokctlConfiguration
apiVersion: kwok.x-k8s.io/v1alpha1
options:
  runtimes: ["nerdctl", "docker"]
```

We use this configuration to switch the order, so we don't need to change these.